code/game/objects/structures/artwork_statue.dm

```diff
@@ -13,6 +14,7 @@
 
 	var/sanity_value = 2 + rand(0,2)
 	AddComponent(/datum/component/atom_sanity, sanity_value, "")
+	price_tag += rand(0,5000)
```

Comment:
Statues are completely useless. Their main value should come from their export price. As such, they should have a pretty high minimum value.
```suggestion
	price_tag += rand(500,5000)
```

Reply:
the minimum price of the statues is 500. this is an extra.